woothemes-sensei.php

```diff
@@ -3,7 +3,7 @@
 Plugin Name: Sensei
 Plugin URI: http://www.woothemes.com/products/sensei/
 Description: A course management plugin that offers the smoothest platform for helping you teach anything.
-Version: 1.9.5
+Version: 1.9.6

```

Comment:
This should be removed from this commit. It will be added later and also represents the wrong version number.